nationalarchives/da-tre-github-actions
# Feature/dte 828 automate jims paresr deploy

As part of the CI/CD of the tna-judgments-parser It will be necessary to 

- get the latest tag of the GitHub repository
-  create ECR repository if not exists
- check if the tag has a matching tag release in ECR
- check latest tag version in ECR
- check if allowed to automatically deploy (dependant on script written by Zazzi)
- build and deploy lambda in container to ECR if required

Several reusable actions have been developed and deployed to top level folders so easier to find and reuse actions
create-ecr-repository
version-deployed
latest-ecr-version
update-allowed

A temporary workflow that can have a manual dispatch have been created to co-ordinate the actions. This will be called from other  workflows or initiated as a chron (TBD)
